api/v1beta1/conditions.go

```diff
@@ -47,4 +47,34 @@ const (
 
 	// ValidateNetworkReadyWaitingMessage not yet ready
 	ValidateNetworkReadyWaitingMessage = "ValidateNetwork not yet ready"
+
+	// InstallOSReadyCondition Status=True condition indicates if the
+	// OS configuration is finished and successful.
+	InstallOSReadyCondition condition.Type = "InstallOSReady"
```

Comment:
The 3 new Ready conditions need to be initialized at https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/controllers/openstackdataplanenode_controller.go#L114

see "Initialize Status" comment in the code

Reply:
done!